# Camera1 Camera2 Merge
string capture_mode
---
int32 error_code
string error_description